### PR TITLE
Update introduction

### PR DESCRIPTION
**Pull request by OR13**
This PR aligns the abstract and intro with generalizing away from specific merkle trees

### CODE REVIEW

**Review by JAG-UK — APPROVED** [review]
LGTM

File: draft-steele-cose-merkle-tree-proofs.md [hunk 1]
```diff
@@ -60,31 +60,28 @@ informative:
...
 through their ability to protect the integrity of batches of documents or collections of statements.
 
-A merkle proof is a path from a leaf to a root in a merkle tree.
+Merkle trees can be constructed from simple operations such as concatenation and digest via a cryptographic hash function,
+however, more advanced constructions enable proofs of different properties of the underlying verifiable data structure.
 
-Merkle proofs can be used to prove a document is in a database (proof of inclusion),
-or that a smaller set of statements are contained in a large set of statements (selective disclosure proofs).
+Verifiable data structure proofs can be used to prove a document is in a database (proof of inclusion),
+that a database is append only (proof of consistency), that a smaller set of statements are contained
```
Comment (robinbryce) [inline draft-steele-cose-merkle-tree-proofs.md:77]: Are consistency proofs on their own sufficent to show a database has been managed in an append only manner ?

Comment (OR13) [reply draft-steele-cose-merkle-tree-proofs.md:77]: The ones from rfc9162 depend on inclusion proofs, but together they prove the append only property. The details are in rfc9162, there is a tree diagram that helps explain this better than the text in our draft.

**Review by henkbirkholz — APPROVED** [review]

File: draft-steele-cose-merkle-tree-proofs.md (hunk 1 above)
Comment (OR13) [reply draft-steele-cose-merkle-tree-proofs.md:77]: Here is an implementation that might be a bit easier to understand:

https://github.com/transmute-industries/rfc9162/blob/main/test/consistencyProof.test.ts

its base on this https://datatracker.ietf.org/doc/html/rfc9162#section-2.1.5

**Pull request merged**